changelogs/unreleased/gh-11115-reconnect-timeout.md

```diff
@@ -0,0 +1,3 @@
+## feature/config
+
+* Added the `replication.reconnect_timeout` option (gh-11115).
```

Comment:
You can also mark it as "part of https://github.com/tarantool/tarantool/issues/10059".